common/test/acceptance/pages/studio/users.py

```diff
@@ -64,7 +64,7 @@ def click_add_button(self):
         """
         Click on the "New Team Member" button
         """
-        self.q(css='.create-user-button').click()
+        self.q(css='.create-user-button').first.click()
```

Comment:
@jzoldak that wait_for is covered in the sequence; however, you bring up a good point. The function would be better off having that logic itself. I've refactored.